packages/extension/src/libs/hoprProvider/index.ts

```diff
@@ -0,0 +1,78 @@
+import RPChSDK, { type Ops } from "@rpch/sdk";
+import { AbstractProvider } from "web3-core";
+import { Result, Error as sdkError } from "@rpch/sdk/build/jrpc";
+
+const ops: Ops = {
+  discoveryPlatformEndpoint:
+    process.env.VUE_APP_DISCOVERY_PLATFORM_API_ENDPOINT || undefined,
+  forceZeroHop: true,
+};
+
+// TODO: Remove after confirmation and testing
+console.log("RPCh: CREATING SDK INSTANCE with OPS ", ops);
+console.log("RPCh: Client ID ", process.env.VUE_APP_RPCH_SECRET_TOKEN);
+
+if (!process.env.VUE_APP_RPCH_SECRET_TOKEN) {
+  throw "MISSING RPCH SECRET TOKEN";
+}
+
+const RPChSdk = new RPChSDK(process.env.VUE_APP_RPCH_SECRET_TOKEN, ops);
```

Comment:
should be a member of the class here

Reply:
It's a question I wanted to raise. 
For every use of `Web3Eth`, a new instance of `RPChProvider` is required, so if an SDK instance would be part of it, we would end up with multiple SDK instances.
I was concerned about performance and network traffic, so I decided to use SDK as a singleton - so having a single instance of SDK across the App.
What do you think about it?

Reply:
Resolved